### PR TITLE
[skip ci] Fixed broken links from #1555

### PR DESCRIPTION
**Pull request by deadjdona**
This PR addresses #1555 

Proposed changes include:
* proper links

### CODE REVIEW

File: doc/user-guide/Features-and-supported-standards.md [hunk 1]
```diff
@@ -73,7 +73,7 @@
 |0270|[XMPP Advanced Server 2010](http://xmpp.org/extensions/xep-0270.html)||
 |0279|[Server IP Check](http://xmpp.org/extensions/xep-0279.html)|`mod_sic`|
 |0280|[Message Carbons](http://xmpp.org/extensions/xep-0280.html)|`mod_carboncopy`|
-|0313|[Message Archive Management](http://xmpp.org/extensions/attic/xep-0313.html)|`mod_mam`|
+|0313|[Message Archive Management](http://xmpp.org/extensions/attic/xep-0313-0.2.html)|`mod_mam`| <!-- xeps are named like xep-0313-0.1.html, -0.2.html, etc. in attic. xep-tool page refers to 0.2 version -->
```
Comment (fenek) [inline doc/user-guide/Features-and-supported-standards.md]: This one should be changed to ordinary link to most recent 0313 version, not the old 0.2 one.

File: doc/user-guide/ICE_tutorial.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ If you don't want to compile this application from source, you can just install
 Right after you start [Mangosta-Android] for the first time, you will need to login to your XMPP server.
 In order to do that, just enter the JID you have created for the phone (*phone@myxmpp.com*), the password (*xmpp_password*) and the server address (*2.2.2.2* or *myxmpp.com* if you've set up the domain to actually point to this IP address), and then confirm by clicking "Enter".
 
-<img src="ICE_tutorial/mangosta_login.png" width="25%">
+<img src="mangosta_login.png" width="25%">
```
Comment (fenek) [inline doc/user-guide/ICE_tutorial.md]: This link was correct.

File: doc/open-extensions/token-reconnection.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ This extension requires the client application to authenticate to the XMPP serve
 
 After that, the following authentications may be done using X-OAUTH SASL mechanism with a token obtained from the server.
 
-To enable the feature, modules `mod_auth_token` and `mod_keystore` have to be enabled on the server. For more details regarding the configuration see [mod_auth_token documentation](../modules/mod_auth_token) and [mod_keystore](module/mod_keystore).
+To enable the feature, modules `mod_auth_token` and `mod_keystore` have to be enabled on the server. For more details regarding the configuration see [mod_auth_token documentation](../modules/mod_auth_token/) and [mod_keystore](../modules/mod_keystore/).
```
Comment (fenek) [inline doc/open-extensions/token-reconnection.md]: Should be `../modules/mod_auth_token.md` and `../modules/mod_keystore.md` actually.

File: doc/user-guide/ICE_tutorial.md (hunk 2 above)
Comment (deadjdona) [reply doc/user-guide/ICE_tutorial.md]: no
https://mongooseim.readthedocs.io/en/latest/user-guide/ICE_tutorial/ICE_tutorial/mangosta_login.png

or add ../ at the beginning, but this is not correct way.

Comment (deadjdona) [reply doc/user-guide/ICE_tutorial.md]: just check the page https://mongooseim.readthedocs.io/en/latest/user-guide/ICE_tutorial/ source

File: doc/open-extensions/token-reconnection.md (hunk 3 above)
Comment (deadjdona) [reply doc/open-extensions/token-reconnection.md]: done

File: doc/user-guide/Features-and-supported-standards.md (hunk 1 above)
Comment (deadjdona) [reply doc/user-guide/Features-and-supported-standards.md]: done

Comment (deadjdona) [reply doc/user-guide/Features-and-supported-standards.md]: it's mentioned on xep-tool page,
maybe it's better to relaunch xep-tool again? is it work?

File: doc/user-guide/ICE_tutorial.md (hunk 2 above)
Comment (fenek) [reply doc/user-guide/ICE_tutorial.md]: Yes, it fixes the image for readthedocs but breaks the page on GitHub further. Most probably the ultimate solution here would be to move the .md file to the same directory as the images.

File: doc/user-guide/Features-and-supported-standards.md (hunk 1 above)
Comment (fenek) [reply doc/user-guide/Features-and-supported-standards.md]: xep-tool will probably need some improvements first.

File: doc/user-guide/ICE_tutorial.md (hunk 2 above)
Comment (deadjdona) [reply doc/user-guide/ICE_tutorial.md]: well, logo works on github, but next 3 images are broken.
on readthedocs logo is broken but 3 images are working.

maybe it's better to fix for readthedocs first, and then fix completely.

**Comment by michalwski** [comment]
I think this PR is no longer relevant. Please re-open or create issue if you think otherwise.

**Pull request closed without merging**